db/database.go

```diff
@@ -94,7 +90,7 @@ func (d *SqlxDatabase) Get(dest interface{}, query string, args ...interface{})
 	return err
 }
 
-func (d *SqlxDatabase) Queryx(query string, args ...interface{}) (rows *Rows, err error) {
+func (d *SqlxDatabase) Query(query string, args ...interface{}) (rows *sqlx.Rows, err error) {
```

Comment:
I think it should return `Rows`.  The fact that it's an actual `sqlx.Rows` is an implementation detail.

Reply:
Aligned to the Tx interface type - but also from what I read "alias" in Go came lake to the party, they do not hide the implementation details like encapsulation.
https://alehatsman.com/posts/golang_type_declaration_abuse.html
https://medium.com/a-journey-with-go/go-aliases-simple-and-efficient-8506d93b079e